lib/router.js

```diff
@@ -5,78 +5,167 @@ const express = require('express');
+ * @param {string} - :id - the id of the record you are trying to find
+ * @returns {object} 200 - JSON object containing the found record
+ * @returns {Error} 500 - Generic error for incorrect param
+ */
 router.get('/:model/:id', handleGetOne);
 
-// TODO: Swagger Comment
+/**
+ * Dynamic route for finding and updating a record
+ * @route PUT /:model/:id
+ * @group Models - Operations performed on the database
+ * @param {string} - :model - the specific model you would like to access
+ * @param {string} - :id - the id of the record you are trying to update
+ * @returns {object} 200 - JSON object containing the updated information as it appears in the database
+ * @returns {Error} 500 - Generic error for incorrect param
+ */
 router.put('/:model/:id', handlePut);
 
-// TODO: Swagger Comment
+/**
+ * Dynamic route for finding a record by it's ID and deleting it from the database
+ * @route DELETE /:model/:id
+ * @group Models - Operations performed on the database
+ * @param {string} - :model - the specific model you would like to access
+ * @param {string} - :id - the id of the record you are trying to delete
+ * @returns {object} 200 - JSON object containing the record that was removed from the database
+ * @returns {Error} 500 - Generic error for incorrect param
+ */
 router.delete('/:model/:id', handleDelete);
 
-// TODO: JSDoc Comment
+/**
+ * Function handGetAll
+ * Handler for routes that are getting all of a collections records
+ * @group CRUD - route handlers
+ * @param {object} req - Object containing the all of the request data sent from the client
+ * @param {object} res - Object containing the response status code as well as JSON data for the count of records and the actual records
+ * @param {function} next - Calls the next function which throws an error if the catch block is hit
+ */
 function handleGetAll(req, res, next) {
   req.model
     .get()
     .then(data => {
       const output = {
         count: data.length,
-        results: data
+        results: data,
       };
       res.status(200).json(output);
     })
     .catch(next);
 }
 
-// TODO: JSDoc Comment
+/**
+ * Function handleGetOne
+ * Handler for routes that are getting a record by it's specified ID, calls the models .get function
+ * @group CRUD - route handlers
+ * @param {object} req - Object containing the all of the request data sent from the client
+ * @param {object} res - Object containing the response status code as well as JSON data for the count of records and the actual records
+ * @param {function} next - Calls the next function which throws an error if the catch block is hit
+ */
 function handleGetOne(req, res, next) {
   req.model
     .get(req.params.id)
     .then(result => res.status(200).json(result[0]))
     .catch(next);
 }
 
-// TODO: JSDoc Comment
+/**
+ * Function handlePost
+ * @group CRUD - route handlers
+ * @param {object} req - Object containing the all of the request data sent from the client
```

Comment:
You could also choose this JSDoc comment to talk about req.body instead of the swagger comment